python/django/v1/ampli_app/util.py

```diff
@@ -0,0 +1,9 @@
+def get_api_key(env_file, key):
```

Comment:
Might be better to use an existing library for this. https://pypi.org/project/python-dotenv/